### PR TITLE
move from_sequence to namespace

### PR DESCRIPTION
**Pull request by MarcoGorelli**
closes https://github.com/data-apis/dataframe-api/issues/162

### CODE REVIEW

File: spec/API_specification/dataframe_api/__init__.py [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
+from ._types import dtype
+
 
 __dataframe_api_version__: str = "YYYY.MM"
 """
 String representing the version of the DataFrame API specification to which the
 conforming implementation adheres.
 """
+
+def from_sequence(sequence: Sequence[object], dtype: dtype) -> Column:
```
Comment (jorisvandenbossche) [inline spec/API_specification/dataframe_api/__init__.py:21]: Should it then be "column_from_sequence" to make it clear it is constructing a Column and not a DataFrame?

**Review by jorisvandenbossche — COMMENTED** [review]
Should we do the same for DataFrame.from_dict?

**Comment by MarcoGorelli** [comment]
good one, thanks for your review - have updated

File: spec/API_specification/dataframe_api/__init__.py [hunk 2]
```diff
@@ -1,14 +1,54 @@
...
+from ._types import dtype
+
 
 __dataframe_api_version__: str = "YYYY.MM"
 """
 String representing the version of the DataFrame API specification to which the
 conforming implementation adheres.
 """
+
+def column_from_sequence(sequence: Sequence[object], dtype: dtype) -> Column:
```
Comment (rgommers) [inline spec/API_specification/dataframe_api/__init__.py:21]: consider making `dtype` keyword-only?

**Review by rgommers — APPROVED** [review]
This looks about ready to me. The `dtype: str` keyword may still change based on gh-155, but that can be done afterwards to not block this PR or other ones - that needs its own PR to deal with `dtype`'s consistently all in one go.

File: spec/API_specification/dataframe_api/__init__.py (hunk 2 above)
Comment (MarcoGorelli) [inline spec/API_specification/dataframe_api/__init__.py:21]:
```suggestion
def column_from_sequence(sequence: Sequence[object], *, dtype: dtype) -> Column:
```

**Comment by MarcoGorelli** [comment]
merging then, thanks for reviewing!

**Pull request merged**